### PR TITLE
release: Print the size of images in a release

### PR DESCRIPTION
**Pull request by smarterclayton**
`--size` shows the user the distribution of content in the release
image and also lays groundwork for verifying the release image by
fetching the referenced images. Add flags from other image commands
to be consistent.

When there is more than one base layer, show an extra column
to indicate which images diverge to assist in correcting build or
inclusion errors

```
Images:
  NAME                                           LAYERS  SIZE MB  UNIQUE MB  BASE
  aws-machine-controllers                             5     91.9       23.6  A
  cli                                                 5     93.3       25.0  A
  cloud-credential-operator                           5     79.5       11.2  A
  cluster-authentication-operator                     5     84.0       15.7  A
  ...
  hyperkube                                           5    117.1       48.8  A
  hypershift                                          5    119.9       51.6  A
  installer                                           5     99.8       31.5  A
  jenkins                                             3    440.9      334.6   B
  jenkins-agent-maven                                 4    441.5      193.8   B
  ...

  2.758GB across 77 layers, 3 different base images
```

### CODE REVIEW

File: pkg/oc/cli/admin/release/info.go [hunk 1]
```diff
@@ -739,6 +797,90 @@ func describeReleaseInfo(out io.Writer, release *ReleaseInfo, showCommit, pullSp
...
+
+				fmt.Fprintf(w, "  %s\t%7d\t%8.1f\t%10.1f\t %s\n", tag.Name, len(image.Layers), float64(size)/1024/1024, float64(unshared)/1024/1024, base)
+			}
+			fmt.Fprintln(w)
+			if len(baseLayer) > 1 {
+				fmt.Fprintf(w, "  %s across %d layers, %d different base images\n", units.HumanSize(float64(totalSize)), len(layerCount), len(baseLayer))
+			} else {
+				fmt.Fprintf(w, "  %s across %d layers\n", units.HumanSize(float64(totalSize)), len(layerCount))
+			}
+
```
Comment (soltysh) [inline pkg/oc/cli/admin/release/info.go]: We should consider splitting those into separate functions for readability.

File: pkg/oc/cli/image/info/info.go [hunk 2]
```diff
@@ -11,6 +11,8 @@ import (
 	"text/tabwriter"
 	"time"
 
+	"github.com/openshift/origin/pkg/oc/cli/image/workqueue"
```
Comment (soltysh) [inline pkg/oc/cli/image/info/info.go]: You're messing our imports :disappointed:

Comment (smarterclayton) [reply pkg/oc/cli/image/info/info.go]: Will fix, I have more improvements coming as I add versioning

**Pull request merged**